addons/web_editor/static/src/js/editor/odoo-editor/src/style.scss

```diff
@@ -300,10 +300,10 @@
         height: 10px;
     }
     ::-webkit-scrollbar-thumb {
-        background: #D3D1CB;
+        background: var(--oePowerbox__ScrollbarThumb-background-color, #D3D1CB);
```

Comment:
Hello @dmo-odoo , we need these lines to correctly handle dark-mode for the editor :)
If we merge them could you eventually forward the change to the lib?

Note that these are CSS variables so if the user don't define them they will juts fallback to the previous values ;)

Reply:
Hello @stefanorigano, the editor is not a separate lib anymore so this is fine :-)

Reply:
thanks @dmo-odoo 👍